### ISSUE TITLE
new release?

### ISSUE THREAD

**Issue by mmoll** [rails/activerecord-session_store#75]
Is there a chance for a new release soon? I mainly want to have a release with 82a7540 to silence the warning/stacktrace with Ruby 2.3.

**Comment by n0nick** [comment]
:+1:

**Comment by benoittgt** [comment]
On my project I just specify the commit into the gemfile to avoid warnings.

```ruby
gem 'activerecord-session_store', git: 'https://github.com/rails/activerecord-session_store', ref: '82a7540'
```

**Comment by mmoll** [comment]
@benoittgt I'm having something similar in my local developement Gemfile, however, that's not feasible for every environment.

**Comment by avvmoto** [comment]
:+1:

**Comment by delphinus35** [comment]
+1

**Comment by benoittgt** [comment]
Please stop with new comment with :+1:. Think about people like me who receive useless email every time... 

Use github recent feature : https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments 

Thanks

**Comment by kwent** [comment]
@sikachu Please ? Anyone trying to use rails `5.0.0.betaX` and using a gem which have `spec.add_dependency "activerecord-session_store` in the `gemspec` can't do anything at all...

**Comment by brauliobo** [comment]
repository collaborators: please add new maintainers so that the project can move forward

**Comment by sikachu** [comment]
I'm sorry for all the frustration. I didn't know the RC would delay for this long, and I think other people were really concentrating on cutting Rails 5.0.0 RC so they overlooked this project.

I'm going to cut a prerelease gem shortly.

Speaking of which — I think this repo collaborators aren't really using this, but are maintaining it because it used to be a part of Rails. Would any of you like to step up and maintain this project? I'll run that by the Rails Core Team to see if they're okay with it.

**Comment by sikachu** [comment]
I just released [v1.0.0.pre](https://github.com/rails/activerecord-session_store/releases/tag/v1.0.0.pre). Please open new issues if you find any compatibility issue with Rails 5.0. Thanks!

**Comment by brauliobo** [comment]
thanks @sikachu, the release is working perfectly
